src/collective/mustread/models.py

```diff
@@ -17,10 +17,12 @@ class MustRead(Base):
     userid = Column(Unicode(255), nullable=True)
     read_at = Column(DateTime, nullable=True)
     deadline = Column(DateTime, nullable=True)
+    scheduled_at = Column(DateTime, nullable=True)
+    scheduled_by = Column(String(255))
     status = Column(String(255), nullable=True)
     uid = Column(String(255))
     type = Column(String(255), nullable=True)
-    title = Column(Unicode(255), nullable=True)
+    title = Column(Unicode(255), nullable=True)  # xxx remove?
     path = Column(String(255), nullable=True)
-    site_name = Column(Unicode(255), nullable=True)
-    info = Column(Unicode(255), nullable=True)
+    site_name = Column(Unicode(255), nullable=True)  # xxx remove?
+    info = Column(Unicode(255), nullable=True)  # xxx remove?
```

Comment:
done.
title was selected in a query, but did not get accessed in the result set